packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -1,7 +1,112 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700');
+@import './variables';
+@import './text_utility';
+
+:root {
+  // stylelint-disable-next-line custom-property-pattern
+  --nav-container-width: 320px;
+}
+
 :local {
-  /* stylelint-disable selector-class-pattern  */
+  @keyframes fadeIn {
```

Comment:
We may be able to use Terra's slide panel so you don't have to do this logic yourself: http://terra-ui.herokuapp.com/components/core/slide-panel